Source/OpenTK/BlittableValueType.cs

```diff
@@ -57,7 +57,11 @@ public static class BlittableValueType<T>
         static BlittableValueType()
         {
             Type = typeof(T);
+#if _NET_CORECLR
+            if (Type.GetTypeInfo().IsValueType && !Type.GetTypeInfo().IsGenericType)
```

Comment:
I've been upgrading a few projects to CoreCLR and some have a minimum requirement of .NET 4.0 thus the #if. If this is not required for OpenTK, then we could simplify the code as you are suggesting.